### PR TITLE
Organizations

### PR DESCRIPTION
**Pull request by Revenaunt**
Add select organization step to on-boarding.

### CODE REVIEW

File: src/main/java/org/tndata/android/compass/util/API.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
  */
 public final class API{
     //Api urls and app configuration
-    public static final boolean STAGING = BuildConfig.DEBUG;
+    public static final boolean STAGING = !BuildConfig.DEBUG;
```
Comment (bradmontgomery) [inline src/main/java/org/tndata/android/compass/util/API.java]: Probably want to revert this change before merging.

Comment (Revenaunt) [inline src/main/java/org/tndata/android/compass/util/API.java]: I always miss that one :disappointed:
